k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientSelectPresenter.kt

```diff
@@ -150,7 +152,7 @@ class RecipientSelectPresenter @Inject constructor(
                     } else {
                         rating
                     }
-                if (isPEpPrivacyProtected && PlanckUtils.isRatingUnsecure(viewRating)
+                if (isPEpPrivacyProtected && viewRating.value == Rating.pEpRatingHaveNoKey.value
```

Comment:
why did we change here the isRatingUnsecure check with just the check of have no key?

Reply:
This is in order to have the count of unsecure recipients in Message Compose screen as only unknown recipients we have no key for. Which means that for now any other unsecure or mistrusted recipient is not taken into account for counting, inviting nor removal.

Reply:
ok I see, thanks for the explaination